### DISCUSSION TITLE
Is there any way to customize the color of the title bar?

### DISCUSSION THREAD

**Discussion by sendudesign** [MicaForEveryone/MicaForEveryone#189] [General]
Whether you choose the light mode or dark mode of the title bar, the UI of the software always looks imperfect.
![Snipaste](https://user-images.githubusercontent.com/118046810/204142137-58a76d72-598c-402e-8293-6accf81035bf.jpg)

**Comment by dongle-the-gadget** [comment]
Can you update your post? You created a discussion prematurely.

**Reply by sendudesign** [reply]
> Can you update your post? You created a discussion prematurely.

ok
